CHANGELOG.md

```diff
@@ -2,13 +2,26 @@
 
 ### Removals
 
-The following contracts were removed:
+The following contracts, libraries and functions were removed:
 
+- `Address.isContract` (because of its ambiguous nature and potential for misuse)
+- `Checkpoints.History`
 - `ERC20Snapshot`
 - `ERC20VotesComp`
+- `ERC165Storage` (in favor of inheritance based approach)
+- `ERC777`
+- `ERC1820Implementer`
 - `GovernorVotesComp`
+- `GovernorProposalThreshold` (deprecated since 4.4)
 - `PaymentSplitter`
-- `TokenTimelock` (removed in favor of `VestingWallet`)
+- `PullPayment`
+- `SafeMath`
+- `SignedSafeMath`
+- `Timers`
+- `TokenTimelock` (in favor of `VestingWallet`)
+- All escrow contracts (`Escrow`, `ConditionalEscrow` and `RefundEscrow`)
+- All CrossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

Comment:
Should we do lowercase "crosschain" following the folder's name?
```suggestion
- All crosschain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

Reply:
```suggestion
- All cross-chain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```